crossplane-packages/helm-incluster.yaml

```diff
@@ -40,3 +40,13 @@ spec:
   package: xpkg.upbound.io/crossplane-contrib/provider-helm:v0.15.0
   controllerConfigRef:
     name: crossplane-provider-helm
+
+---
+
+apiVersion: helm.crossplane.io/v1beta1
```

Comment:
Necessary because the app config chart needs to be applied to the current cluster becuase it relies on the terraform provider